.gitmodules

```diff
@@ -12,3 +12,12 @@
 [submodule "vendor/uutf"]
 	path = vendor/uutf
 	url = https://github.com/dune-universe/uutf.git
+[submodule "vendor/octavius"]
+	path = vendor/octavius
+	url = https://github.com/ocaml-doc/octavius.git
+  ignore = dirty
+[submodule "vendor/omd"]
+	path = vendor/omd
+	url = https://github.com/ulugbekna/omd.git
+	branch = ocaml-lsp
```

Comment:
```suggestion
	url = https://github.com/ocaml/omd.git
```

Reply:
Now it uses dune.

Reply:
The current implementation does use dune but lacks functionality such as `to_markdown: t -> string` function. 
I was hoping to merge this and then work on a PR to Omd (I expected a PR to Omd to make longer work and review time).

Reply:
Oh, I missed it. @nojb is taking care of it these days, so it might be a good idea to contact them.

Reply:
Sorry coming late to the discussion; we are getting a release ready which breaks a bunch of things, so it would be useful to hear what you are using and what you would need so that we can take it into account.

Reply:
OK, I see from reading the code that you are generating Markdown AST programmatically and would like to convert it to string.

To be honest it would be rather easy (easier?) to generate the markdown on your side directly instead of going through Omd. It is the parsing of Markdown that is tricky :)

But I *think* it wouldn't be hard for us to include this functionality in the new release; I will look into it.